src/main/webapp/js/components/fields/field-config-modal.jsx

```diff
@@ -243,6 +247,7 @@ class FieldConfigModal extends React.Component {
 
 const mapStateToProps = (state, props) => ({
   fieldConfig: getFieldConfigById(state, props),
+  entities: state.entity[props.entityType],
```

Comment:
for field config it will be a little different, right now you're getting the list of entities, not the field configs for this entity type, it should be `state.fieldConfig[props.entityType]`